library/couchbase

```diff
@@ -1,11 +1,19 @@
 Maintainers: Couchbase Docker Team <docker@couchbase.com> (@cb-robot)
 GitRepo: https://github.com/couchbase/docker
 
-Tags: latest, enterprise, 7.0.0, enterprise-7.0.0
+Tags: latest, enterprise, 7.0.1, enterprise-7.0.1
+GitCommit: f412542829939319b5e722fa384c60b68e765bce
+Directory: enterprise/couchbase-server/7.0.1
+
+Tags: community, community-7.0.1
+GitCommit: f412542829939319b5e722fa384c60b68e765bce
+Directory: community/couchbase-server/7.0.1
+
+Tags: 7.0.0, enterprise-7.0.0
 GitCommit: a4a4b016732b4ea3a42ea115f481c66ba69b980d
 Directory: enterprise/couchbase-server/7.0.0
 
-Tags: community, community-7.0.0
+Tags: community-7.0.0
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)